src/rcc.rs

```diff
@@ -28,7 +30,31 @@ impl RccExt for RCC {
             crrcr: CRRCR { _0: () },
             cfgr: CFGR {
                 hclk: None,
-                hsi48: false,
+                usb48: false,
```

Comment:
Would renaming usb48 to usbRngSDMMC48 be better ? As this is that same clock line?
![clocktree](https://user-images.githubusercontent.com/314381/51427554-96a96000-1bf9-11e9-80c2-4d8fd4dfe166.png)

Reply:
I think my other comments address this.